src/styles/blocks/header.scss

```diff
@@ -38,5 +38,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color #{$smoothly-transition};
+    transition-duration: #{$smoothly-transition};
```

Comment:
you don't need to specify transition duration as you've already done it in the transition